pylab/core/models.py

```diff
@@ -56,3 +56,26 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('event-details', args=[self.starts.year, self.starts.month, self.starts.day, self.slug])
+
+
+class Attendance(models.Model):
```

Comment:
```Event.attendees``` would be a Queryset, and you could apply a filter based on answer. (At least I guess so).